react-charts/area-chart/step4.md

```diff
@@ -1,61 +1,59 @@
-Now we'll add multiple datasets to the area chart...
+Add tooltips to the area chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartArea
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-  ]} 
-/&gt;
+&lt;Chart
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `containerComponent` property to that section</strong>
+
+The `containerComponent` property takes a component instance, which will be used to create a container element for standalone charts.
+
+Add the `ChartVoronoiContainer` component which adds the ability to associate a mouse position with the data point(s) closest to it. This is useful for adding a tooltip.
+
+It should look like this:
 
 <pre class="file" data-target="clipboard">
-&lt;ChartGroup&gt;
-  &lt;ChartArea
-    data={[
-      { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-      { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-      { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-      { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-    ]}
-    interpolation="basis"
-  /&gt;
-  &lt;ChartArea 
-    data={[
-      { name: &#39;Dogs&#39;, x: &#39;2015&#39;, y: 2 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2016&#39;, y: 3 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2017&#39;, y: 4 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2018&#39;, y: 5 },
-      { name: &#39;Dogs&#39;, x: &#39;2019&#39;, y: 6 }
-    ]}
-    interpolation="basis"
+containerComponent={
+  &lt;ChartVoronoiContainer
   /&gt;
-  &lt;ChartArea 
-    data={[
-      { name: &#39;Birds&#39;, x: &#39;2015&#39;, y: 1 }, 
-      { name: &#39;Birds&#39;, x: &#39;2016&#39;, y: 2 }, 
-      { name: &#39;Birds&#39;, x: &#39;2017&#39;, y: 3 }, 
-      { name: &#39;Birds&#39;, x: &#39;2018&#39;, y: 2 },
-      { name: &#39;Birds&#39;, x: &#39;2019&#39;, y: 4 }
-    ]}
-    interpolation="basis"
+}
+</pre>
+
+<strong>Note:</strong> There are no visible changes at this stage.
+
+4) <strong>Add a `constrainToVisibleArea` property inside of the `ChartVoronoiContainer`.</strong>
+
+The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG.

```

Comment:
We are using a little bit of jargon here.
Victory has a pretty good description in their docs:
```suggestion
The `constrainToVisibleArea` property determines whether to coerce tooltips so that they fit within the visible area of the chart.
```